src/integrant/core.cljc

```diff
@@ -193,6 +199,17 @@
        (map ref-key)
        (remove #(find-derived config %))))
 
+(defn- invalid-composite-keys [config]
+  (->> (keys config)
+       (filter vector?)
+       (filter (comp not composite-key?))))
```

Comment:
This could be written as `(remove composite-key?)` to make things clearer.

I think you could also put it all on one line for vertical brevity:

```clojure
(defn- invalid-composite-keys [config]
  (->> config keys (filter vector?) (remove composite-key?)))